R/derive_vars_aage.R

```diff
@@ -37,12 +37,36 @@
 #'
 #' @param unit *Deprecated*, please use `age_unit` instead.
 #'
-#' @details The age is derived as the integer part of the duration from start to
-#'   end date in the specified unit. When 'years' or 'months' are specified in the `out_unit`
-#'   parameter, because of the underlying `lubridate::time_length()` function that is used
-#'   here, results are calculated based on the actual calendar length of months or years
-#'   rather than assuming equal days every month (30.4375 days) or every year (365.25 days).
+#' @param type lubridate duration type.
 #'
+#'   See below for details.
+#'
+#'   Default: `"duration"`
+#'
+#'   Permitted Values: `"duration"`, `"interval"`
+#'
+#' @details The duration is derived as time from start to end date in the
+#'   specified output unit. If the end date is before the start date, the duration
+#'   is negative. The start and end date variable must be present in the specified
+#'   input dataset.
+#'
+#'   The [lubridate](https://lubridate.tidyverse.org/) package calculates two
```

Comment:
Is there a way for us inherit this from the source? derive_vars_duration??  Just so if we update it in the other function it populates here.

Reply:
I think we should add a `roxygen_*()` function for this (once we have introduced this concept). As far as I know inheritance is only possible for `@param` tags.